### PR TITLE
Move e2e debugging guide to separate page

### PR DESCRIPTION
**Pull request by U8NWXD**
This starts to build up a section in the sidebar for debugging guides, as described in my proposal. I will be adding guides for other tests to this section in future PRs. This PR doesn't have much in the way of content changes except for a few spelling fixes.

### CODE REVIEW

**Comment by U8NWXD** [comment]
@DubeySandeep PTAL

**Review by krishnarao22 — APPROVED** [review]
Looks good!

File: Debug-end-to-end-tests.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+CircleCI allows debugging using SSH. For details, please read [this](https://circleci.com/docs/2.0/ssh-access-jobs/#steps). Debugging with SSH only reruns that particular job, so it is a great way to rerun a passing test to see if it flakes without rerunning all the tests.
+
+## Downloading screenshots
+
+We capture screenshots of failing tests. On CircleCI, these are available under the `Artifacts` tab of the failure log page. You may also want to reference the [CircleCI artifacts documentation](https://circleci.com/docs/2.0/artifacts/). On GitHub Actions, look for an `Artifacts` link in the upper right where you can download a zip file of the screenshots.
+
+Here's an example of what artifacts on CircleCI look like:
+
+![CircleCI artifacts page with hyperlinks to screenshots and reports](https://user-images.githubusercontent.com/19878639/111242142-f1ba2000-85d4-11eb-8bf1-66cfbbf71975.png)
```
Comment (DubeySandeep) [inline Debug-end-to-end-tests.md:73]: Should we update this with Github-Action's artifacts tab?

**Review by DubeySandeep — COMMENTED** [review]
Thanks for the review @krishnarao22!

@U8NWXD I've left a  few comments, lets merge this PR once those comments are resolved!

File: Debug-end-to-end-tests.md (hunk 1 above)
Comment (U8NWXD) [reply Debug-end-to-end-tests.md]: done

File: Debug-end-to-end-tests.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+7. The control will stop at the point where the debugger statement was added. You can now choose to inspect elements, log variables in the test, or add more break-points.
+
+## Rerunning with SSH
+
+CircleCI allows debugging using SSH. For details, please read [this](https://circleci.com/docs/2.0/ssh-access-jobs/#steps). Debugging with SSH only reruns that particular job, so it is a great way to rerun a passing test to see if it flakes without rerunning all the tests.
+
+## Downloading screenshots
+
+We capture screenshots of failing tests. On CircleCI, these are available under the `Artifacts` tab of the failure log page. You may also want to reference the [CircleCI artifacts documentation](https://circleci.com/docs/2.0/artifacts/). On GitHub Actions, look for an `Artifacts` link in the upper right where you can download a zip file of the screenshots.
```
[the remark this answers is not in the document]
Comment (U8NWXD) [reply Debug-end-to-end-tests.md]: done

File: Debug-end-to-end-tests.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+   ![Inspect Page](https://user-images.githubusercontent.com/11008603/88563290-714bac80-d04f-11ea-8b36-fc43c66d6e3d.png)
+
+6. A Chrome dev tools instance will open up and the e2e test should start executing in a new window.
+
+7. The control will stop at the point where the debugger statement was added. You can now choose to inspect elements, log variables in the test, or add more break-points.
+
+## Rerunning with SSH
+
+CircleCI allows debugging using SSH. For details, please read [this](https://circleci.com/docs/2.0/ssh-access-jobs/#steps). Debugging with SSH only reruns that particular job, so it is a great way to rerun a passing test to see if it flakes without rerunning all the tests.
```
[the remark this answers is not in the document]
Comment (U8NWXD) [reply Debug-end-to-end-tests.md]: nope! I removed the CircleCI reference

**Comment by U8NWXD** [comment]
@DubeySandeep PTAL

File: Debug-end-to-end-tests.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
+## Table of contents
+
+* [Introduction](#introduction)
+* [Using the debugger](#using-the-debugger)
+* [Rerunning with SSH](#rerunning-with-ssh)
+* [Downloading screenshots](#downloading-screenshots)
+* [Downloading screen recordings](#downloading-screen-recordings)
```
Comment (DubeySandeep) [inline Debug-end-to-end-tests.md:7]: Let's update the table of contents with the new changes and make sure all the links are working :)

**Review by DubeySandeep — APPROVED** [review]
Feel free to merge this PR once the last comment is resolved!

**Comment by U8NWXD** [comment]
Removed seanlip from reviewers list (per decision by @DubeySandeep) since this PR doesn't change much content

File: Debug-end-to-end-tests.md (hunk 4 above)
Comment (U8NWXD) [reply Debug-end-to-end-tests.md]: done

**Pull request merged**